frontend/src/repository/exams.js

```diff
@@ -1,13 +1,46 @@
 import axios from 'axios';
+import moment from "moment";
 
 export const getExams = async () => {
     const { data } = await axios.get('http://localhost:4000/api/exams');
     return data;
 };
 
+export const getExam = async (id) => {
+    if (!id) return;
+    const { data } = await axios.get(`http://localhost:4000/api/exams/${id}`);
+    return data;
+};
+
+export const getExamsWithScaleAndTime = async () => {
+    const {data} = await axios.get('http://localhost:4000/api/exams');
+    let examScale = 0;
+    let examTime = 0;
+    for (let i = 0; i < data.length; i++) {
+        for (let j = 0; j < data[i].exercices.length; j++) {
+            for (let k = 0; k < data[i].exercices[j].questions.length; k++) {
```

Comment:
triple for loop, yum 😋

Reply:
@ValentinChion champion here! :p

Reply:
@ValentinChion , I think you can achieve that in a cleaner/easier way using reduce. 

Apply reduce and forEach. I can provide the code if needed.